Justfile

```
@@ -33,6 +33,11 @@ start-oracle oracle-name:
   cd "{{root-dir}}/apps/oracles/{{oracle-name}}"
   RUST_LOG=trigger=info "${SPIN:-spin}" build --up
 
+get-machine-outpath commit machine:
+  #!/usr/bin/env bash
+  set -euo pipefail
+  nix eval --raw "github:blocksense-network/blocksense?rev={{commit}}#nixosConfigurations.{{machine}}.config.system.build.toplevel.outPath"
```

* consider renaming `get-machine-outpath` to `eval-machine`, as we will probably add `build-machine` and `start-machine` in the future
* Make the `commit` parameter last and make it optional
* Allow building with local (uncommitted) changes 


```suggestion
eval-machine machine commit="working-tree":
  #!/usr/bin/env bash
  set -euo pipefail
  if [[ commit = "working-tree" ]; then
    nix eval --raw "{{ root-dir }}#nixosConfigurations.{{machine}}.config.system.build.toplevel.outPath"
  else
    nix eval --raw "github:blocksense-network/blocksense?rev={{commit}}#nixosConfigurations.{{machine}}.config.system.build.toplevel.outPath"
  fi
```